### PR TITLE
[doc] update runtime-filter.md

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
update runtime-filter.md

### CODE REVIEW

**Comment by shasevendow** [comment]
Does the 2.1 version also need to be updated?

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query/join-optimization/runtime-filter.md [hunk 1]
```diff
@@ -138,15 +138,13 @@ set runtime_filter_type=7;
 
 - **IN or Bloom Filter**: 根据右表在执行过程中的真实行数，由系统自动判断使用 IN predicate 还是 Bloom Filter
   
-  - 默认在右表数据行数少于 102400 时会使用 IN predicate（可通过 session 变量中的`runtime_filter_max_in_num`调整），否则使用 Bloom filter。
+  - 默认在右表数据行数少于 runtime_filter_max_in_num 时会使用 IN predicate，否则使用 Bloom filter。
```
Comment (HappenLee) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query/join-optimization/runtime-filter.md:141]: 115 line had delete the line `runtime_filter_max_in_num`, maybe add again

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**